src/app/http.ts

```diff
@@ -0,0 +1,62 @@
+import fetch from 'node-fetch';
+
+const host = process.env.MM_URL || 'http://localhost:8065';
+const pathPrefix = `${host}/plugins/com.mattermost.apps`;
+
+const pathInstall = `${pathPrefix}/install`;
+const pathSubscribe = `${pathPrefix}/subscribe`;
+const pathConnectedInstall = `${pathPrefix}/connected_install`;
+
+// http constains methods for handling requests to and from the
+// mattermost-plugin-apps API
+class MattermostPluginHTTP {
+    async installApp() {
+        // console.log(JSON.stringify(dialog));
+        // const res = await fetch(u, {method: 'POST', body: JSON.stringify(dialog)}).then(r => r.json());
+        // return res.json();
+        const opts = {
+            method: 'POST',
+            headers: {
+                Authorization: 'Basicjunkencoded',
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify('junk'),
+        };
+
+        const r = await fetch(pathInstall, opts);
```

Comment:
This way the caller can catch an error, or inspect the fetch response

```suggestion
        return fetch(pathInstall, opts);
```

Reply:
file has been removed.